.github/scripts/compare-benchmarks.py

```
@@ -0,0 +1,429 @@
+
+        # Distinguish CPU vs Memory benchmarks by presence of Gen data
+        if result.gen0 > 0 or result.gen1 > 0 or "Memory" in name:
+            bench_data["gen0"] = result.gen0
+            bench_data["gen1"] = result.gen1
+            bench_data["gen2"] = result.gen2
+            memory_benchmarks.append(bench_data)
+        else:
+            cpu_benchmarks.append(bench_data)
+
+    baseline = {
+        "date": datetime.now().isoformat(),
```

Ruff (0.14.3)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)